types/permissions.ts

```diff
@@ -23,7 +24,7 @@ export enum DbRolePermissions {
   create_thread_on_realm = "create_thread_on_realm",
   access_locked_boards_on_realm = "access_locked_boards_on_realm",
```

Comment:
At some point we may want a "delete_boards_on_realm" permission so we don't have to assign "delete_board" separately for each board, but that's a design decision for future us.